### PR TITLE
Represent execution times in the job's timezone

### PR DESCRIPTION
**Pull request by yvanoers**
Implements #526.
This applies the job's timezone to the execution's start and end times when executions get unmarshalled, so everything that uses them afterwards, including the dashboard and the notifier should present the times in the job's timezone.

@victorcoder, let me know what you think.

### CODE REVIEW

File: dkron/store.go [hunk 1]
```diff
@@ -627,7 +627,16 @@ func (s *Store) Shutdown() error {
 	return s.db.Close()
 }
 
-func (s *Store) unmarshalExecutions(items []*kv, stopWord string) ([]*Execution, error) {
+func (s *Store) unmarshalExecutions(items []*kv, jobName string) ([]*Execution, error) {
+	// Get the job's timezone
+	var jobLocation *time.Location
+	if job, err := s.GetJob(jobName, nil); err == nil {
```
Comment (victorcoder) [inline dkron/store.go]: Could we avoid going to the store while unmarshaling?

Comment (yvanoers) [reply dkron/store.go]: I'm not too thrilled about that either. Avoiding it might require a lot of plumbing - but I'll see what can be done about it.

Comment (yvanoers) [reply dkron/store.go]: How about this:
Commit 1d614c3 refactors things so the timezone gets passed down to unmarshalExecutions.
Now, only the dashboard goes to the store to retrieve the timezone.

**Comment by victorcoder** [comment]
This diff is also not ok, a clean up is needed.

**Pull request closed without merging**